common/protos/proto/user.proto

```diff
@@ -178,4 +178,6 @@ message UserStageInfo {
   CopyOptions copy_options = 5;
 
   string comment = 6;
+
+  string created_on = 7;
```

Comment:
To keep backwards compatible,it seems  need 'Option' here

Reply:
I don't like the idea to store a time string in metasrv.

How about converting to [Timestamp](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/timestamp.proto) or UNIX timestamp as u64.

Reply:
Good suggestion, will try it :D